node/wrappers.js

```diff
@@ -7,34 +7,60 @@ const isObject = require("../lib/utils.js").isObject;
 const utils = require("./utils.js");
 const reifySemVer = utils.getReifySemVer();
 const reifyVersion = reifySemVer.version;
+const hasOwn = Object.prototype.hasOwnProperty;
 
-function addWrapper(func, wrapper) {
-  const reified = func.reified;
-  if (typeof reified.wrappers[reifyVersion] !== "function") {
-    reified.versions.push(reifyVersion);
-    reified.wrappers[reifyVersion] = wrapper;
+function getSymbolForKey(key) {
+  return Symbol.for("reifyWrapperMap$" + key);
```

Comment:
Don't worry about it for this PR though.
I'll do a cleanup pass and tie everything together :bowtie: